basics/keeping-up-to-date/upgrade-module/upgrade-module-internal-behavior.md

```diff
@@ -80,6 +80,18 @@ Classes are gathered by responsibilities:
 - Classes interacting with PrestaShop core
 - Tasks (Can be considered as controllers for upgrade, rollback etc.)
 
+## Upgrade modules from local source
+
+During the autoupgrade process, in the "modules upgrade" step, we check for each module if there is a new version available in the PrestaShop Marketplace
```

Comment:
```suggestion
During the upgrade process, we check for new module versions in the PrestaShop Marketplace. If there is one, we download it and then upgrade it.
```

Reply:
Is it only about the native modules? How does it work if you have a paid module and you are not logged in to your Addons account?

Reply:
I don't change the modules upgrade process

I guess if we have a paid module and the user is not logged to addons/account, the new version download fails

Reply:
If there is a zip given, I don't check the Addons connection